src/sentry/apidocs/public_exclusion_list.py

```diff
@@ -742,6 +743,7 @@
     OrganizationIssuesCountEndpoint,
     OrganizationGroupIndexStatsEndpoint,
     GroupActivitiesEndpoint,
```

Comment:
instead of modifying this file please set `private = True` in the endpoint. thanks!